--- mame-153/src/osd/sdl/sdl.mak	2014-07-13 20:59:51.184583518 +0200
+++ mame-153.patch/src/osd/sdl/sdl.mak	2014-07-13 21:05:51.454147085 +0200
@@ -486,30 +486,31 @@
 
 DEFS += -DSDLMAME_UNIX
 
-ifndef NO_USE_QTDEBUG
-MOCTST = $(shell which moc-qt4 2>/dev/null)
-ifeq '$(MOCTST)' ''
-MOCTST = $(shell which moc 2>/dev/null)
-ifeq '$(MOCTST)' ''
-$(error Qt's Meta Object Compiler (moc) wasn't found!)
-else
-MOC = @$(MOCTST)
-endif
-else
-MOC = @$(MOCTST)
-endif
-# Qt on Linux/UNIX
-QMAKE = $(shell which qmake-qt4 2>/dev/null)
-ifeq '$(QMAKE)' ''
-QMAKE = $(shell which qmake 2>/dev/null)
-ifeq '$(QMAKE)' ''
-$(error qmake wasn't found!)
-endif
-endif
-QT_INSTALL_HEADERS = $(shell $(QMAKE) -query QT_INSTALL_HEADERS)
-INCPATH += -I$(QT_INSTALL_HEADERS)/QtCore -I$(QT_INSTALL_HEADERS)/QtGui -I$(QT_INSTALL_HEADERS)
-LIBS += -L$(shell $(QMAKE) -query QT_INSTALL_LIBS) -lQtGui -lQtCore
-endif
+NO_USE_QTDEBUG = 1
+#ifndef NO_USE_QTDEBUG
+#MOCTST = $(shell which moc-qt4 2>/dev/null)
+#ifeq '$(MOCTST)' ''
+#MOCTST = $(shell which moc 2>/dev/null)
+#ifeq '$(MOCTST)' ''
+#$(error Qt's Meta Object Compiler (moc) wasn't found!)
+#else
+#MOC = @$(MOCTST)
+#endif
+#else
+#MOC = @$(MOCTST)
+#endif
+## Qt on Linux/UNIX
+#QMAKE = $(shell which qmake-qt4 2>/dev/null)
+#ifeq '$(QMAKE)' ''
+#QMAKE = $(shell which qmake 2>/dev/null)
+#ifeq '$(QMAKE)' ''
+#$(error qmake wasn't found!)
+#endif
+#endif
+#QT_INSTALL_HEADERS = $(shell $(QMAKE) -query QT_INSTALL_HEADERS)
+#INCPATH += -I$(QT_INSTALL_HEADERS)/QtCore -I$(QT_INSTALL_HEADERS)/QtGui -I$(QT_INSTALL_HEADERS)
+#LIBS += -L$(shell $(QMAKE) -query QT_INSTALL_LIBS) -lQtGui -lQtCore
+#endif
 
 LIBGL = -lGL
 
